tables/de-g0-core.uti

```diff
@@ -184,26 +185,28 @@ always  (?         2356-6-26
 endnum  !          6-235
 midword !          6-235
 always  \s!        0-6-235
-noback always  !          235
+always  !          235
+
+#word ' 6
 
 midword :          6-25
 
 midword (          6-2356
-midnum  (          6-2356-3456
+midnum  (          6-2356
 endnum  (          6-2356
-noback always  (          2356
+always  (          2356
 
 midword )          6-2356
-midnum  )          6-2356-3456
+midnum  )          6-2356
 endnum  )          6-2356
-noback always  )          2356
+always  )          2356
 
 midword [          6-6-2356
-midnum  [          6-6-2356-3456
-endnum  [          6-6-2356
+midnum  [          6-2356
+endnum  [          6-2356
 midword ]          6-6-2356
-midnum  ]          6-6-2356-3456
-endnum  ]          6-6-2356
+midnum  ]          6-2356
+endnum  ]          6-2356
```

Comment:
These do not only remove the `3456` but also a `6`. Maybe something that needs to be double checked?

Reply:
I double checked. In fact that was a problem. This might have to be fixed differently (make the nonumsign work properly), but for now the `midnum` and `endnum` rules are ok